__docs__/wonder-blocks-icon/accessibility.mdx

```diff
@@ -32,15 +32,11 @@ More information about all these points can be found in the
 
 ### Demo: Contrast
 
-<Canvas>
-  <Story of={AccessibilityStories.IconContrast} />
-</Canvas>
+<Canvas of={AccessibilityStories.IconContrast} />
```

Comment:
**note:** Regarding the docs changes. Turns out the Storybook syntax changed but we didn't get any error indicating that.

Now, `Canvas` doesn't render children - it uses the `of` prop to know what story to show and without that, will always render the default story from the imported stories.

I have fixed all uses including those that used additional properties on Canvas to show source code.

Reply:
> Thanks for fixing this. Was the only way to see that it was broken to look at these pages in Storybook?

Yes, unfortunately. I thought at first I had broken the hooks, and had to dig into the stories more before I discovered this issue.